lxd/instance_sftp.go

```diff
@@ -67,6 +67,7 @@ func instanceSFTPHandler(d *Daemon, r *http.Request) response.Response {
 		return response.SmartError(err)
 	}
 	if client != nil {
+		client = client.UseProject(projectName)
```

Comment:
Good catch, I'll update the tests to cover this.

Reply:
Also, I'm surprised that `client, err := cluster.ConnectIfInstanceIsRemote(d.cluster, projectName, instName, d.endpoints.NetworkCert(), d.serverCert(), r, instanceType)` doesn't return a client preconfigured with the `projectName` argument, that would make sense and be helpful right?

Reply:
I remember thinking that too last time I hit that issue :)

I don't remember if I tracked it down and figured it would break something or didn't have time and left it the way it was.